### PR TITLE
(maint) Use yum to install build dependencies on AIX

### PR DESCRIPTION
**Pull request by geoffnichols**
As a POC, update the AIX 6.1 platform config to bootstrap and use yum
to install build dependencies.

### CODE REVIEW

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by McdonaldSeanp** [comment]
I don't know if the yum.sh script we put on pl-build-tools is the same, but If it is I would prefer that we perform the steps ourselves (or make our own scripts).

the yum.sh "easy install" script attempts to download everything from the IBM hosted servers, and there really isn't an option in the script to change that. That means we lose all of the benefits of our internal artifactory mirrors, which I would rather not do.

I'm assuming this isn't a critical path PR or anything (and if it's just to POC yum-on-aix that's cool). So this comment is mostly for posterity

File: configs/components/_base-ruby-augeas.rb [hunk 1]
```diff
@@ -17,9 +17,13 @@
 pkg.build_requires "ruby-#{ruby_version}"
 pkg.build_requires "augeas"
 
-pkg.environment "PATH", "$(PATH):/opt/pl-build-tools/bin:/usr/local/bin:/opt/csw/bin:/usr/ccs/bin:/usr/sfw/bin"
+if settings[:use_system_build_tools]
```
Comment (McdonaldSeanp) [inline configs/components/_base-ruby-augeas.rb]: there shouldn't really be a need to specify different PATHs when using system tools.

We should just leave the PATH alone and if the pl-build-tools dir isn't there nothing will care anyway

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix-6.1-ppc

**Comment by geoffnichols** [comment]
jenkins please test this on aix61-POWERfa

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix-6.1-ppc

**Comment by geoffnichols** [comment]
jenkins please test this with puppet-agent on aix61-POWERfa

**Pull request closed without merging**